src/Raven.Server/Documents/Queries/Parser/QueryExpression.cs

```diff
@@ -150,8 +150,7 @@ public void ToJavaScript(string query, string alias, TextWriter writer)
                     break;
                 case OperatorType.Method:
                     var method = Extract(query, Field.TokenStart, Field.TokenLength, Field.EscapeChars);
-                    var methodType = QueryMethod.GetMethodType(method);
-                    if (methodType == MethodType.Id)
+                    if (string.Equals(method, "id", StringComparison.OrdinalIgnoreCase))
```

Comment:
Not a good idea, you need to do that _iff_ there are no arguments.

Consider:
```
from Users u
load u.Manager as m
select {
   ManagerId: id(m)
}
```

Reply:
Done